### PR TITLE
COMP: Setting CMP0020 to NEW

### PR DESCRIPTION
**Pull request by adamrankin**
References issue [https://issues.slicer.org/view.php?id=4396](https://issues.slicer.org/view.php?id=4396)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@ endif()
 foreach(p
   CMP0048 # CMake 3.0
   CMP0054 # CMake 3.1
+  CMP0020 # CMake 3.0.2
```
Comment (msmolens) [inline CMakeLists.txt]: I believe these versions are when the policies were introduced, so it should be 2.8.11 for CMP0020.

Comment (adamrankin) [inline CMakeLists.txt]: Yup! Misread the documentation. Fixing it now.

**Comment by lassoan** [comment]
It'll be great to get rid of this warning!

I've integrated this and performed a clean build but it is still printed when building CTK and Slicer. Did this change suppress the warning for you?

**Comment by adamrankin** [comment]
It suppressed it only for Slicer. CTK will have to set their own rules.

**Comment by lassoan** [comment]
OK, I see the same then. It's in CTK and EMSegment only.

**Comment by lassoan** [comment]
Committed r26168. Thank you for your contribution.

**Comment by adamrankin** [comment]
Just fyi, github is telling me "Closed with unmerged commits
This pull request is closed, but the adamrankin:4396-set-qtmain-policy branch has unmerged commits."

**Comment by lassoan** [comment]
This repository is a mirror. The changes were committed in r26168 in SVN.

**Comment by pieper** [comment]
Will you make a PR for ctk too?  😄

**Pull request closed without merging**